include/wil/resource.h

```diff
@@ -4616,6 +4616,33 @@ namespace wil
     }
 #endif // WIL_ENABLE_EXCEPTIONS
 
+    inline wil::unique_variant make_variant_bstr_nothrow(PCWSTR source) WI_NOEXCEPT

```

Comment:
Only comment I have - and it's mostly to solicit opinions from others - is that checking for failure isn't as simple as using something like `operator bool` on the return value, so I wonder if changing the return type to be `HRESULT` and have the `unique_variant` be an out param might be more "friendly."

On the flip side, if it's an out param, the value would always need to be cleared, even if it was just constructed, so my preference is to probably keep it this way.